### ISSUE TITLE
Bilboussa City Jail

### ISSUE THREAD

**Issue by IanarT** [Hatchie-Studio/lotstracker#42]
@On Bilboussa, the only way to get out of the jail is to buy your way out (unless cutdoor works). I propose that there is a way to get out otherwise. Doesn't have to be pickable or doorbashable, but maybe a hidden exit. Somewhat like the stock jail on... I think its Byss?

Not to mention that, there is a thief in the jail cell with you stealing your credits. Makes it hard to buy your way out. I feel like at the very least the thief should be removed from the cell.

**Comment by michaelirick** [comment]
I'll look into it, thanks.

**Comment by Thanatoslots** [comment]
I don't really like the hidden exit thing. Though we should remove any of the ones in the prison.

**Comment by IanarT** [comment]
Yeah, I don't really agree with hidden exits either, but I honestly don't think there should be absolutely no way to get out of a place like that. Just my opinion.
